package.json

```diff
@@ -10,78 +10,39 @@
     "dev": "vite",
     "start": "vite dist --host",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test": "vitest",
     "prettier": "prettier --write src/."
   },
-  "jest": {
-    "globals": {
-      "ts-jest": {
-        "useESM": true
-      }
-    },
-    "verbose": true,
-    "testMatch": [
-      "<rootDir>/**/__tests__/*.test.ts"
-    ],
-    "transform": {
-      "^.+\\.svelte$": [
-        "svelte-jester",
-        {
-          "preprocess": true
-        }
-      ],
-      "^.+\\.ts$": "ts-jest",
-      "^.+\\.js$": "babel-jest"
-    },
-    "moduleFileExtensions": [
-      "js",
-      "ts",
-      "svelte"
-    ],
-    "setupFilesAfterEnv": [
-      "<rootDir>/src/setup_tests.ts",
-      "jest-expect-message"
-    ],
-    "transformIgnorePatterns": []
-  },
   "devDependencies": {
     "@babel/preset-env": "^7.23.5",
     "@bulatdashiev/svelte-slider": "^1.0.3",
-    "@smui/dialog": "^7.0.0-beta.3",
```

Comment:
@smui/dialog was entirely unused so I've just removed it

Reply:
Might have been an old import. Thank you for the thoroughness :)